### ISSUE TITLE
Add version number to a parameter set

### ISSUE THREAD

**Issue by valentinsulzer** [FaradayInstitution/BPX#25]
Add version number to a parameter set, separate from (in addition to) the BPX version number.

This would allow us to fix mistakes or inaccuracies in parameter sets while maintaining old versions (and hence not changing any previous results/publications that have been obtained using an old version).

It can probably just be a single version number, doesn't need to be semantic versioning.

**Comment by rtimms** [comment]
Add a new field to the `Header`, e.g.

```python3

    version: str = Field(
        None,
        alias="Version",
        example="0.1.1",
        description="BPX file version",
    )
```

@Ubham16

**Comment by rtimms** [comment]
Fixed by #63
